components/core/layout/TwoColWrapper.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div class="two-col-layout">
         <div class="left-col">
-            <slot></slot>
+            <slot name="left-col"></slot>
```

Comment:
This is not relevant to the PR and do break the other page (`2022/<lang>/registration/financial-aid`). Please revert the change.